tests/baselines/reference/truthinessCallExpressionCoercion2.errors.txt

```diff
@@ -87,8 +86,6 @@ tests/cases/compiler/truthinessCallExpressionCoercion2.ts(112,14): error TS2774:
         // error
         typeof window !== 'undefined' && window.console &&
             ((window.console as any).firebug || (window.console.exception && window.console.table));
-                                                 ~~~~~~~~~~~~~~~~~~~~~~~~
-!!! error TS2774: This condition will always return true since the function is always defined. Did you mean to call it instead?

```

Comment:
This error disappeared as a consequence of the temporary fix I mentioned in https://github.com/microsoft/TypeScript/pull/42835/files#r577513285. It will appear again once that is resolved.